src/Builder/ArchiveBuilderHelper.php

```diff
@@ -59,13 +62,15 @@ public function isSkippable(PackageInterface $package): bool
 
         $names = $package->getNames();
 
-        if ($this->archiveConfig['whitelist'] && !$this->isOneOfNamesInList($names, $this->archiveConfig['whitelist'])) {
+        $whiteListIsEmpty = 0 === count($this->archiveConfig['whitelist']);
+        if (!$whiteListIsEmpty && !$this->isOneOfNamesInList($names, $this->archiveConfig['whitelist'])) {
```

Comment:
Do you really need a variable?

```suggestion
        if (0 !== count($this->archiveConfig['whitelist']) && !$this->isOneOfNamesInList($names, $this->archiveConfig['whitelist'])) {
```

Reply:
I believe that makes it more readable with no performance cost

Reply:
Nah, I reread it and yeah the variable doesn't help with that. I've updated the pr